src/main/java/net/neoforged/neoforge/client/extensions/IMinecraftExtension.java

```diff
@@ -41,4 +44,16 @@ default void popGuiLayer() {
     default Locale getLocale() {
         return self().getLanguageManager().getJavaLocale();
     }
+
+    /**
+     * Returns the clients {@link FlagManager}.
+     * <p>
+     * This instance should receive updates from the server as needed.
+     * Prefer using {@link ILevelReaderExtension#getModdedFlagManager()} where possible.
+     *
+     * @return client sided {@link FlagManager}.
+     */
+    default FlagManager getModdedFlagManager() {
```

Comment:
This method can just be a patch in `Minecraft` itself, also specifying it in this extension interface provides no benefit